docs/CONAN_USER_DOCUMENTATION.md

```diff
@@ -1,3 +1,14 @@
-# Documentation
+# How to use Conan with NXRM
 
-Coming soon!
+1. Install Conan (if on OSX you can do this via ```brew install conan```)
+2. Create the conan repositories in NXRM (e.g. conan-hosted)
+3. Add the conan security realm to NXRM: Login as an admin, go to settings > security > realms and add 
+   ```Conan Bearer Token Realm```
+4. (Optional) Conan will already have a remote configured so if you want it to exclusively use NXRM you will need to remove that 
+   remote first via ```conan remote remove conan-center```
+5. Add your newly created repository/repositories to conan 
+   ```conan remote add nxrm-conan-hosted http://localhost:8081/repository/conan-hosted/ false```. Note: the false at the
+   end relates to whether you want to use SSL or not. If you have configured NXRM to have SSL enabled you can switch
+   this to true.
```

Comment:
Q: Is it "along with adjusting the URL" or will it work with true and the http url?

Reply:
No idea 😄 I'll leave that to the reader to work out